hack/common/consumer.sh

```diff
@@ -72,7 +72,7 @@ spec:
             - "--store-path=/store"
             - "--transport-host=$transport_host"
             - "--http-event-publishers=$http_event_publishers"
-            - "--api-port=9095"
+            - "--api-port=9043"
```

Comment:
for v2 there should not be a cloud-event-proxy sidecar. Please refer to https://github.com/redhat-cne/cloud-event-proxy/tree/main/examples/manifests/v2

Reply:
I don't understand. The sidecar has been removed.